src/css/main.css

```diff
@@ -0,0 +1,118 @@
+body {
+  background: #e8e8e8;
+  height: 700px;
+}
+.main-container {
+  min-height: 100%;
+}
+.navbar-default,
+.btn-default {
+  background-color: #cd5c5c !important;
```

Comment:
Refrain from using `!important` as much as possible. Check https://developer.mozilla.org/en/docs/Web/CSS/Specificity for more details